src/main/kotlin/no/nav/familie/ks/sak/integrasjon/oppgave/OppgaveService.kt

```diff
@@ -36,12 +43,27 @@ class OppgaveService(private val integrasjonClient: IntegrasjonClient) {
     fun hentOppgaver(finnOppgaveRequest: FinnOppgaveRequest): FinnOppgaveResponseDto =
         integrasjonClient.hentOppgaver(finnOppgaveRequest)
 
+    fun hentOppgaverSomIkkeErFerdigstilt(behandling: Behandling): List<Dboppgave> {
+        return oppgaveRepository.findByBehandlingAndIkkeFerdigstilt(behandling)
+    }
```

Comment:
Hadde vært nice hvis vi kunne ha vært mer konsekvent med single expression functions.
Ser at vi gjør det for de andre metodene som eksisterer i denne fila, så tenker vi kan gjøre det. samme her.

```suggestion
    fun hentOppgaverSomIkkeErFerdigstilt(behandling: Behandling): List<Dboppgave> =
        oppgaveRepository.findByBehandlingAndIkkeFerdigstilt(behandling)
```

Reply:
Det samme gjelder resten av metodene i denne klassen.

Reply:
fikset det